services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -59,170 +60,172 @@
 import uk.gov.moj.sdt.services.utils.api.IMessagingUtility;
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
 /**
  * Test class for the RetryMessageAlreadySentService.
  *
  * @author Manoj Kulkarni
  */
-public class RetryMessageAlreadySentServiceTest extends AbstractSdtUnitTestBase {
+@ExtendWith(MockitoExtension.class)
+class RetryMessageAlreadySentServiceTest extends AbstractSdtUnitTestBase {
     /**
      * Mocked Individual Request Dao object.
      */
+    @Mock
     private IIndividualRequestDao mockIndividualRequestDao;
 
     /**
      * The mocked ICacheable reference to the global parameters cache.
      */
+    @Mock
     private ICacheable mockCacheable;
 
     /**
      * Mocked messaging utility reference.
      */
+    @Mock
     private IMessagingUtility mockMessagingUtility;
 
     /**
      * RetryMessageAlreadySentService instance to be tested.
      */
     private RetryMessageAlreadySentService messageTaskService;
 
+    private static final String MAX_FORWARDING_ATTEMPTS = "MAX_FORWARDING_ATTEMPTS";
+    private static final String TEST_1 = "TEST_1";
+    private static final String TEST_2 = "TEST_2";
+    private static final String FORWARDED = "Forwarded";

```

Comment:
These constant names aren't very descriptive!  "TEST_1" and "TEST_2" are used as SDT request references so constants can be named SDT_REQUEST_REF and DEAD_LETTER_SDT_REQUEST_REF.  "Forwarded" is a request status so the constant can be named STATUS_FORWARDED.

Reply:
done

Reply:
FORWARDED constant hasn't been renamed.  Were you intending to do this or do you think it's OK as it is?

Reply:
renamed to STATUS_FORWARDED

Reply:
Confirmed rename of constant.